lib/workers/types.ts

```diff
@@ -16,7 +16,9 @@ import type {
 import type { PlatformPrOptions } from '../platform/types';
 import type { File } from '../util/git/types';
 import type { MergeConfidence } from '../util/merge-confidence';
-import type { ChangeLogResult } from './pr/changelog/types';
+import type { ChangeLogNotes, ChangeLogResult } from './pr/changelog/types';
+
+export type BranchUpgrade = Release & { releaseNotes?: ChangeLogNotes };
```

Comment:
```suggestion
export type BranchUpgrade = ChangeLogRelease & Release;
```

@rarkins do you have a better name than `BranchUpgrade` 🤔

Reply:
`ReleaseWithNotes` ?

Reply:
Updated suggestion 🙃

Reply:
done